### ISSUE TITLE
Improve the visualisation API doc

### ISSUE THREAD

**Issue by Diachk** [Linkurious/documentation#144]
There are a few issue in the API documentation for visualisation:

- [ ] There are no examples of `create visualisation` data model
- [ ] The parameters of `update visualisation` are partially defined

https://doc.linkurio.us/server-sdk/latest/apidoc/#api-Visualizations

**Comment by fantandrea** [comment]
@francescoinfante can you confirm this part of the doc is in the API src code? Are you handling it?

**Comment by francescoinfante** [comment]
@fantandrea yes, API doc is in the server code next to the API declaration itself.
